staging/src/k8s.io/sample-controller/main.go

```diff
@@ -65,8 +65,8 @@ func main() {
 		kubeInformerFactory.Apps().V1().Deployments(),
 		exampleInformerFactory.Samplecontroller().V1alpha1().Foos())
 
-	go kubeInformerFactory.Start(stopCh)
-	go exampleInformerFactory.Start(stopCh)
+	kubeInformerFactory.Start(stopCh)
+	exampleInformerFactory.Start(stopCh)
 
 	if err = controller.Run(2, stopCh); err != nil {
```

Comment:
Shouldn't this be run as a go routine though?

Reply:
I think it is okay, the main func will stop at this method as it essentially runs the controller and blocks until the stopCh is closed.